src/state/application/updater.ts

```diff
@@ -119,7 +119,13 @@ export default function Updater(): null {
   }, [dispatch, debouncedState.chainId])
 
   useEffect(() => {
-    if (!account || !library?.provider?.request || !library?.provider?.isMetaMask) {
+    // @ts-ignore
+    const isCbWalletDappBrowser = window?.ethereum?.isCoinbaseWallet
+    // @ts-ignore
+    const isWalletlink = !!window?.WalletLinkProvider || !!window?.walletLinkExtension
+    const isCbWallet = isCbWalletDappBrowser || isWalletlink
+    const isMetamaskOrCbWallet = library?.provider?.isMetaMask || isCbWallet
+    if (!account || !library?.provider?.request || !isMetamaskOrCbWallet) {
       return
     }
     switchToNetwork({ library })
```

Comment:
i'm not sure we need to do this test anymore, @JFrankfurt this was just for older versions of metamask right?

Reply:
fwiw, I am seeing some dapps just assume that wallets support 3085 requests. might be better to just remove this check so every other wallet doesn't have to add special flags here?

Reply:
This check was for older versions of mm, yeah. My gut says it's still necessary (since Brave still ships w/ an old version.)